tests/backup/backup_helper.go

```diff
@@ -3996,6 +3999,34 @@ func IsClusterPresent(clusterName string, ctx context.Context, orgID string) (bo
 	return false, nil
 }
 
+// GetConfigObj reads the configuration file and returns a BackupCloudConfig object.
+func GetConfigObj() (*backup.BackupCloudConfig, error) {
+	var config *backup.BackupCloudConfig
+	cmList, err := core.Instance().ListConfigMap("default", meta_v1.ListOptions{})
+	log.FailOnError(err, fmt.Sprintf("Error listing ConFigMaps in default namespace"))
+	found := false
```

Comment:
By default, in go treats variables as false.
You can just declare `var found bool`